Gemfile

```diff
@@ -23,6 +23,9 @@ gem 'mysql2'
 # for observing records
 gem 'rails-observers'
 
+# for rating
+gem 'ratyrate', github: 'wazery/ratyrate', branch: 'master'
```

Comment:
Using the master branch is as simple as using this line above in the Gemfile. It would be great if we can discuss this more on the issue itself rather than here. :slightly_smiling_face: